src/datastore/pg-write-store.ts

```diff
@@ -1683,6 +1683,10 @@ export class PgWriteStore extends PgStore {
         bns_subdomains_imported = ${configState.bns_subdomains_imported},
         token_offering_imported = ${configState.token_offering_imported}
     `;
+    await this.notifier?.sendBnsImport({
+      bnsNamesOnchainImported: configState.bns_names_onchain_imported,
+      bnsSubdomainsImported: configState.bns_subdomains_imported,
+    });
```

Comment:
Now that I think about it, since you're calling this on `updateConfigState` I think you should rename your update to `configStateUpdate` and add the `tokenOfferingImported` property as well.

That way we'll have a more generalized update when any config has changed.